packages.json

```diff
@@ -2334,7 +2334,6 @@
   "https://github.com/liveui/apicore.git",
   "https://github.com/liveui/configure.git",
   "https://github.com/LiveUI/ErrorsCore.git",
-  "https://github.com/LiveUI/FluentTestTools.git",
```

Comment:
I'm not sure why this package is flagged for removal. The URL is live and `swift package dump-package` succeeds.

Reply:
I'm guessing it was this error that made us skip it:

```
package 2300 ...
ERROR: Error Domain=NSURLErrorDomain Code=-1001 "(null)"
```

Reply:
Looks like this is a time out. We must not let that skip the package, should only skip on definite 403/404.